packages/x-bus/src/x-bus.types.ts

```diff
@@ -1,15 +1,6 @@
 import { Dictionary } from '@empathyco/x-utils';
 import { Observable, Subject } from 'rxjs';
 
-/**
- * Alias representing a
- * {@link https://developer.mozilla.org/en-US/docs/Web/API/setTimeout#return_value | timeoutID}
- * value.
- *
- * @internal
- */
-export type TimeoutId = number;
```

Comment:
What do you think about keep using this type, setting it to:

```
/**
 * Alias representing a
 * {@link https://developer.mozilla.org/en-US/docs/Web/API/setTimeout#return_value | timeoutID}
 * value.
 * It is a `number` in browser/jsdom environments, or a `Timeout` object in `node` environments.
 *
 * @internal
 */
export type TimeoutId = ReturnType<typeof setTimeout>;
```

This way it should be compatible with both environments and we can use `setTimeout` instead of `window.setTimeout`.
This type is being used in some utils of `x-components`, `x-components/src/utils/debounce`, for example.
Maybe we can move it to the `x-types` package, to make it reusable across packages.

Reply:
@LuisMartinez15, I think @ajperezbau already took a look, and I'm testing it now too.